tests/phpunit/tests/blocks/editor.php

```diff
@@ -18,6 +18,8 @@ public function set_up() {
 
 		parent::set_up();
 
+		remove_action( 'wp_print_styles', 'print_emoji_styles' );
```

Comment:
Why is this needed here? Now that the function is deprecated, we probably shouldn't need to remove this in tests? The same applies below as well.

Reply:
@felixarntz I get a deprecated message warning in the tests, making the tests fail.